### PR TITLE
Adds the ClientInfo object + fields to TokenResponse

### PR DESCRIPTION
**Pull request by iambmelt**
Creates the POJO for `ClientInfo`, adds it as a field on `AzureActiveDirectoryTokenResponse`

### CODE REVIEW

**Comment by iambmelt** [comment]
Also adds fields `ext_expires_on` -> 'Resiliency' and `not_before` token issuance tracking

File: common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryTokenResponse.java [hunk 1]
```diff
@@ -85,4 +101,58 @@ public String getIdToken() {
     public void setIdToken(String idToken) {
```
Comment (shoatman) [inline common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryTokenResponse.java]: Unrelated to this PR... just didn't catch this on the last one.  We should consider moving IDToken to base TokenResponse... http://openid.net/specs/openid-connect-core-1_0.html#TokenResponse

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryTokenResponse.java]: Roger that 👍

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/azureactivedirectory/AzureActiveDirectoryTokenResponse.java]: Resolved in 9110b3a84e222b8463e7fb8b93ecf42771423acc

**Pull request merged**